src/App.tsx

```diff
@@ -1,24 +1,287 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState, useEffect, useMemo, useCallback,
+} from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  updateTodoTitle,
+  updateTodoStatus,
+} from './api/todos';
 
-const USER_ID = 0;
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { getVisibleTodos, getTodosId } from './utils/utils';
+import { IdsContext } from './utils/Context/IdsContext';
+
+const USER_ID = 10631;
+
+export const filterOptions = ['All', 'Active', 'Completed'];
```

Comment:
instead array make enum for filter options. it will increase readibility and avoid set filter by filterOptions index.